frontend/src/metabase/common/components/EntityPicker/components/ResultItem/ResultItem.styled.tsx

```diff
@@ -10,16 +11,18 @@ export const ChunkyListItem = styled.button<{
   cursor: pointer;
 
   background-color: ${({ isSelected }) =>
-    isSelected ? color("brand") : "white"};
+    isSelected ? "var(--mb-color-brand)" : color("white")};
 
   color: ${({ isSelected }) =>
     isSelected ? color("white") : color("text-dark")};
 
   &:hover {
     ${({ isSelected }) =>
       !isSelected
-        ? `background-color: ${color("brand-lighter")};
-      color: ${color("text-dark")};`
+        ? css`
+            background-color: var(--mb-color-brand-lighter);
+            color: ${color("text-dark")};
+          `
         : ""}
```

Comment:
I wonder if we can do `!isSelected && css...`, to shorten this. If not that's fine.